install/kubernetes/cilium/templates/hubble-relay-client-tls-secret.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if and (.Values.hubble.relay.enabled) (.Values.hubble.tls.enabled) (hasKey .Values.hubble "listenAddress") }}
+{{- if not (hasKey .Values "quickInstall") }}
```

Comment:
nit: I might be missing a bit of context here, but I find the naming here a bit too generic. Maybe we can make this more specific to the Hubble TLS certs? 

There are many legitimate use-cases where a user wants to deploy Cilium+Hubble with TLS enabled, but provide the TLS certs manually that are unrelated to quickInstall. So I would also be fine to name this `.Values.hubble.tls.noCreateSecrets` or something similar.

Reply:
> There are many legitimate use-cases where a user wants to deploy Cilium+Hubble with TLS enabled, but provide the TLS certs manually (totally unrelated to quick-install.yaml).

Thanks for pointing this out. There are so many boolean values in these files that I go confused. I don't even need the `quickInstall` variable. I just needed to set `hubble.tls.enabled=false`.